kotlinx-coroutines-core/common/src/JobSupport.kt

```diff
@@ -19,6 +19,7 @@ import kotlin.jvm.*
  * @param active when `true` the job is created in _active_ state, when `false` in _new_ state. See [Job] for details.
  * @suppress **This is unstable API and it is subject to change.**
  */
+@OptIn(InternalForInheritanceCoroutinesApi::class)
```

Comment:
is there a reason why `JobSupport` isn't annotated with `InternalCoroutinesApi` at all?

Reply:
Yes: it's deprecated with an `ERROR`.